### PR TITLE
Feature/service endpoint additions

### PR DESCRIPTION
**Pull request by Rovack**
Add support for updating hours and languages of a service.

### CODE REVIEW

File: src/services/services.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import { getDayOfWeekInteger } from '../utils/times';
```
Comment (Rovack) [inline src/services/services.js]: Sorry about the name of this file... :P

File: src/services/services.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+const { sequelize } = models;
+
+const updateHours = async (service, hours, t) => {
+  await models.RegularSchedule.destroy({ where: { service_id: service.id }, transaction: t });
+
+  await Promise.all(hours.map(hoursPart => models.RegularSchedule.create({
+    service_id: service.id,
+    weekday: getDayOfWeekInteger(hoursPart.weekday),
+    opens_at: hoursPart.opensAt,
+    closes_at: hoursPart.closesAt,
```
Comment (nlaz) [inline src/services/services.js]: How do we support 24/7 versus non-24/7 services? https://www.figma.com/file/fdzJjvkSugVGnUa3zNdvMNsU/StreetTeam_source_of_truth?node-id=282%3A4818

Comment (nlaz) [reply src/services/services.js]: Alright sounds good to me 👍

**Comment by Rovack** [comment]
Well, opening and closing for each day shouldn't be a problem, as each record has a weekday field.
For multiple opening and closing times within a single day, I think that should be fine too: we can just have multiple records with the same weekday - there's no unique constraint on it.
So, you could send e.g.:
```
[
  { weekday: 'Sunday', opensAt: '08:00', closesAt: '12:00'  },
  { weekday: 'Sunday', opensAt: '16:00', closesAt: '19:00'  },
  { weekday: 'Monday', opensAt: '08:00', closesAt: '15:00'  },
]
```

**Pull request merged**